lms/djangoapps/discussion_api/api.py

```diff
@@ -276,7 +286,12 @@ def get_thread_list(request, course_key, page, page_size, topic_id_list=None, te
         "per_page": page_size,
         "text": text_search,
     }
+
     text_search_rewrite = None
+
+    if view:
+        query_params[view] = "true"
```

Comment:
There's a test that confirms if it is not valid. 

https://github.com/edx/edx-platform/pull/8571/files#diff-36265ec8db98beae9aa3738e6311f285R146

I didn't add tests to that verifies the correct view choices because I feel like those tests would just be verifying that ChoiceField works. Then again, it might be worthwhile to test correct values.